File: app/src/main/java/com/dvalic/appaudiclass/repositorys/InterfazFragments.java
package com.dvalic.appaudiclass.repositorys;

import android.os.Bundle;

import kotlin.Unit;
import kotlin.reflect.KVisibility;

public interface InterfazFragments {

    //Todo MainMenu
    void showMainMenufragment();
    void showModelsfragment();


    //Todo Utility
    void showBars(Boolean visibility);
    void showWebPage(String url);
    void showSnackbar(String url, int type);
    void showWebView(Bundle bundle);
    void showPdf(Bundle bundle);
    void showDialog(String title,String description,String textPositiveButton,String textNegativeButton);
}

File: app/src/main/java/com/dvalic/appaudiclass/repositorys/InterfazFragments.kt
package com.dvalic.appaudiclass.repositorys

import android.os.Bundle

interface InterfazFragments {

    //Todo MainMenu
    fun showMainMenufragment()
    fun showModelsfragment()
    fun showVersionsFragment(bundle: Bundle?)

    //Todo Utility
    fun showBars(visibility: Boolean)
    fun showWebPage(url: String?)
    fun showSnackbar(url: String?, type: Int)
    fun showWebView(bundle: Bundle?)
    fun showPdf(bundle: Bundle?)
    fun showDialog(
        title: String?,
        description: String?,
        textPositiveButton: String?,
        textNegativeButton: String?,
        actionPositive: Unit?,
        actionNegative: Unit?
    )
}